content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Upgrade to Prisma 3'
+metaTitle: 'Upgrade to Prisma 3'
+metaDescription: 'Guides on how to upgrade to Prisma 3'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+To upgrade from version 2.x to 3.x, you need to update both the `prisma` and `@prisma/client` packages. Both the `prisma` and `@prisma/client` packages install with a caret `^` in their version number to safe guard against breaking changes.
```

Comment:
I would introduce with something explaining that Prisma 3 comes with a set of breaking changes, detailed below, that people should check before upgrading, and then explain how to upgrade.